### PR TITLE
moved logs from drawer menu to the settings

### PR DESCRIPTION
**Pull request by AndyScherzinger**
As discussed in #1432 the Logs menu item from the drawer has been moved to the settings screen.

@jancborchardt I moved it to the "more" section instead of the "advanced" section - is this correct or should I move it to the "advanced section? 
device-2016-03-22-145222

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@jancborchardt I can also move the settings item in the drawer to the bottom, or separate it with a line like ather apps do it. Whatever floats your boat :wink:

**Comment by AndyScherzinger** [comment]
Google apps just separate it with a line:
Gmail
device-2016-03-22-150703
Drive
device-2016-03-22-150808
Hangout
device-2016-03-22-150738

Dropbox - top right
device-2016-03-22-150846

Amaze file manager like Google bat as a sticky footer
device-2016-03-22-150908

**Comment by AndyScherzinger** [comment]
@jancborchardt having done a local implementation using the Support Library for the drawer's content implementation the menu items would look like this...
device-2016-03-22-200828
with an item "active":
device-2016-03-22-200848
The difference is very, very subtile since the unactive grey is quite close to the primary color (dark blue).
The Drawer width has also increased to our actual one since this is the standard with in material...

What do you think? Moving on with this has it's benefits since checked/unchecked state, items via native menu.xml files/groups would then be available :)

**Comment by jancborchardt** [comment]
I'd say either we stick it to the bottom or we leave it as it is. The line just increases the focus on settings more, which is not what we want. ;)

**Comment by AndyScherzinger** [comment]
Okay :9 So when the reliable uploads are in place (on master) with the new drawer it would then look like this:
device-2016-03-22-224635

**Comment by AndyScherzinger** [comment]
I removed the new drawer implementation here and started a new PR for the drawer over here: #1559 
So this PR is basically a move of the logs to the settings screen and a removal from the drawer so you can forget about the latest screenshots since this won't be part of this PR anymore.

**Comment by AndyScherzinger** [comment]
@rperezb @jancborchardt @davivel @tobiasKaminsky This is DONE and can be reviewed...

**Comment by tobiasKaminsky** [comment]
:+1: code is fine!

**Comment by AndyScherzinger** [comment]
pinging @davivel and @malkomich minor change, freshly rebased, community reviewed, resolving  #1432 raised by @jancborchardt

**Comment by jancborchardt** [comment]
cc @davivel @MTrichards @rperezb please review.

**Comment by rperezb** [comment]
@jancborchardt @AndyScherzinger we will check this after the team week thx

**Comment by jabarros** [comment]
:+1:

File: res/values/setup.xml [hunk 1]
```diff
@@ -53,6 +53,7 @@
     <bool name="imprint_enabled">false</bool> 
     <bool name="recommend_enabled">true</bool>
     <bool name="feedback_enabled">true</bool>
+    <bool name="logger_enabled">true</bool>
```
Comment (davivel) [inline res/values/setup.xml]: Why is this here?

**Comment by davivel** [comment]
:-1: ; please, see my question

File: res/values/setup.xml (hunk 1 above)
Comment (AndyScherzinger) [inline res/values/setup.xml]: For the same reason the other are there ;) --> with this flag you can decide if the log part will be rendered in the settings screen or not. We can take it out if you want to. I just thought like with imprint, feedback etc. you would like to be able to have to feature toggle for logging also since I discussed the move to the settings screen with @jancborchardt and we agreed that after moving it to the settings screen it will also be visible in the official release.

**Comment by AndyScherzinger** [comment]
:wink: please see my answer :smiley:

File: res/values/setup.xml (hunk 1 above)
Comment (davivel) [inline res/values/setup.xml]: Thanks, I understand now. I totally oversaw the intention of making the menu available in the official release of the app, not only in debug mode. In this case, having this option here makes a lot of sense.

The point is, I'm not so sure we should have this is the official app. I will follow this up in #1432

**Comment by davivel** [comment]
Thanks, @AndyScherzinger .

Blocked until a decision is taken about make the logs available in release mode or not.

cc @rperezb

**Comment by jancborchardt** [comment]
Ok – let’s make it available in debug mode / beta only maybe for now. And just move the place of it. We’re doing 2 separate steps here (moving, and making it visible for normal users), that’s the problem.

**Comment by davivel** [comment]
Thanks, @jancborchardt.

Proposal: let's keep the option in ```settings.xml```, but with default value to ```false```. Besides, let's modify the condition in ```Preferences.java``` so that the option is available if ```(loggerEnabled || BuildConfig.DEBUG)```. Seems to me more comfortable.

@AndyScherzinger , what do you think?

**Comment by AndyScherzinger** [comment]
Yeah, sounds good to me! I can make the change during the next days! :)

**Comment by AndyScherzinger** [comment]
@davivel added the changes as discussed :)

**Comment by davivel** [comment]
Great job :)

:+1:

@jesmrec , your turn.

**Comment by jesmrec** [comment]
@davivel @AndyScherzinger @jabarros 

Approved 👍

**Comment by jabarros** [comment]
Cool, so it is prepared to be merged :)
@AndyScherzinger could you rebase it on master and merge it by yourself? Thanks!!

**Comment by AndyScherzinger** [comment]
DONE! :+1:

**Comment by rperezb** [comment]
Thanks!
Good to know that there is a new variable on the setup file ☺️
